documents/converters.go

```diff
@@ -329,30 +331,66 @@ func ToClientCollaboratorAccess(ca CollaboratorsAccess) (*documentpb.ReadAccess,
 }
 
 // DeriveResponseHeader derives common response header for model
-func DeriveResponseHeader(model Model) (*documentpb.ResponseHeader, error) {
+func DeriveResponseHeader(tokenRegistry TokenRegistry, model Model) (*documentpb.ResponseHeader, error) {
 	cs, err := model.GetCollaborators()
 	if err != nil {
 		return nil, errors.NewTypedError(ErrCollaborators, err)
 	}
 
-	rcs, wcs := ToClientCollaboratorAccess(cs)
-	// TODO(ved): we need to update log for NewCoreDocumentWithCollaborators and PrepareNewVersion. Please don't do this without further discussion since this affects consensus validations.
-	//author, err := model.Author()
-	//if err != nil {
-	//	return nil, err
-	//}
-	//
-	//time, err := model.Timestamp()
-	//if err != nil {
-	//	return nil, err
-	//}
+	author := ""
+	a, err := model.Author()
+	// we ignore error here because it can happen when a model us first created but its not anchored yet
```

Comment:
```suggestion
	// we ignore error here because it can happen when a model is first created but its not anchored yet
```

Reply:
In which case why do we need the err check? why not just assign author=a.String()

Reply:
error check is just there to warn. But I realise we can directly us a.String in either case.

Reply:
shit I realise I removed the warn anyway